usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -268,16 +268,15 @@ def handle(self, *args, **options):
 
             # Create the base award, create actions, then tally the totals for the award
             award_field_map = {
-                # Adding _id on to create the FK
-                "awarding_agency_id": "awarding_agency_code",
-                "funding_agency_id": "funding_agency_code",
                 "description": "award_description",
                 "type": "assistance_type"
             }
 
             award_value_map = {
                 "period_of_performance_start_date": format_date(row['period_of_performance_star']),
                 "period_of_performance_current_end_date": format_date(row['period_of_performance_curr']),
+                "awarding_agency": Agency.objects.filter(pk=row["awarding_agency_code"]).first(),
```

Comment:
I think the agency primary key was changed from CGAC to an autoincrementing ID. So this filter would need to change to querying by CGAC. Although, looking at the updates to the model, it seems like it was changed to an autoincrementing id but the CGAC field was not added to replace it. Also the comment was not updated, meaning it's suuuper confusing :). I can push a commit to update the agency model in a few minutes.